source/module_base/test/spherical_bessel_transformer_test.cpp

```diff
@@ -43,7 +43,7 @@ class SphericalBesselTransformTest : public ::testing::Test
     void TearDown();
 
     /// Gets the maximum absolute element-wise difference between two arrays
-    double max_diff(const int sz, const double* const arr1, const double* const arr2);
+    static double max_diff(int sz, const double* arr1, const double* arr2);
 
     int sz_max = 10000;         ///< size of each buffer
```

Comment:
const int sz_max?

Reply:
a const qualifer has been added in the latest commit